pyproject.toml

```diff
@@ -28,6 +28,7 @@ test = [
     "flake8-bugbear",
     "black",
     "scikit-learn",
+    "tensorflow >=2.5.0",
```

Comment:
This is heavy requirement but i have no idea how to circumvent it :/

Sadly still the best way to get the dataset.

Reply:
See below comment.
```suggestion
```

Reply:
```suggestion
    "tensorflow >=2.0.0",
```